src/mergerepo_c.c

```diff
@@ -115,6 +115,8 @@ struct CmdOptions {
 
     gboolean version;
     char **repos;
+    char *repo_prefix;
+    char *repo_prefix_url;
```

Comment:
Could you replace these (and also the options) to followin?:
* repo_prefix -> repo_prefix_search
* repo_prefix_url -> repo_prefix_replace